source/libs/utils.ts

```diff
@@ -88,6 +88,10 @@ export const getOwnerAndRepo = (): {
 	return {ownerName, repoName};
 };
 
+export function getForkedRepo(): string | undefined {
```

Comment:
This is ambiguous. Is it the repo that was forked or the fork itself?

The correct term would be "source repo". Perhaps `getForkSourceRepo` would be the correct term.

Can you change this name everywhere? Including `const forkedRepo = ...`

Reply:
I thought so as well, but I trusted the existing naming (the function existed before, I just moved it to utils).

Yes, I'll take care of it when I'm going to handle #2934.

Reply:
Hm, there's already a function with that name in the `forked-to` feature.

```ts
const getForkSourceRepo = (): string => getForkedRepo() ?? getRepoURL();
```

Reply:
How about
- `getParentRepo`: only works if there’s a parent)
- `getSourceRepo`: always gets the source repo, whether it’s the parent or the current

Reply:
Sound's good to me. I'll do some refactoring of these, but I think I'll open a separate PR for that.